internal/install/execution/recipe_var_provider.go

```
@@ -180,8 +184,16 @@ func varFromEnv() types.RecipeVars {
 	envDownloadURL := os.Getenv("NEW_RELIC_DOWNLOAD_URL")
 	if envDownloadURL != "" {
 		URL, err := url.Parse(envDownloadURL)
-		if err == nil && URL.Scheme == "https" && strings.HasSuffix(URL.Host, "newrelic.com") {
-			downloadURL = envDownloadURL
+		if err == nil && URL.Scheme == "https" {
```

Should we return the error if `err != nil`?